mkdocs.yml

```diff
@@ -15,9 +15,18 @@ nav:
   - Home: index.md
   - Getting Started:
     - Installation: GettingStarted/install.md
-  - Examples:
+    - Dependencies: GettingStarted/dependencies.md
+    - Building Applications with SELF: GettingStarted/building-with-self.md
+  - Tutorials:
     - Burgers Equation:
-      - Traveling Shock : Examples/BurgersEquation1D/TravelingShock.md
+      - Traveling Shock : Tutorials/BurgersEquation1D/TravelingShock.md
+  #    - Create your own model: Tutorials/CreateYourOwnModel.md
+  - Models:
+    - Viscous Burgers Equation: Models/burgers-equation-model.md
```

Comment:
Consider adding commented line for generic DG for consistency with other upcoming pages

```suggestion
    - Viscous Burgers Equation: Models/burgers-equation-model.md
#    - Generic DG Models: Models/generic-dg-models.md
```